src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -34,12 +34,13 @@
     private Stage primaryStage;
     private Logic logic;
 
-    private Image logoPicture = new Image(this.getClass().getResourceAsStream("/images/covigent_logo.png"));
-
+    private Image logoImage = new Image(this.getClass().getResourceAsStream("/images/Logo.png"));
```

Comment:
Maybe we can standardize the casing for file names (i.e. `logo.png`).

Reply:
okay